src/helpers/files.ts

```diff
@@ -63,14 +71,16 @@ const matchesRewrite = (file, rewrites) => {
   return matchesRedirect(file, rewrites.beforeFiles)
 }
 
-exports.matchesRedirect = matchesRedirect
-exports.matchesRewrite = matchesRewrite
-exports.matchMiddleware = matchMiddleware
-exports.stripLocale = stripLocale
-exports.isDynamicRoute = isDynamicRoute
-
 // eslint-disable-next-line max-lines-per-function
-exports.moveStaticPages = async ({ netlifyConfig, target, i18n }) => {
+export const moveStaticPages = async ({
+  netlifyConfig,
+  target,
+  i18n,
+}: {
+  netlifyConfig: NetlifyConfig
+  target: 'server' | 'serverless' | 'experimental-serverless-trace'
```

Comment:
[sand] Could have a type for this instead

Reply:
👏🏻 
<img width="214" alt="image" src="https://user-images.githubusercontent.com/213306/145361426-5875be5a-83fe-45b2-b42d-9d3631ea745b.png">